deployment/documentation/Dockerfile.documentation

```diff
@@ -2,4 +2,6 @@ FROM squidfunk/mkdocs-material
 
 RUN pip install --upgrade pip
 
+RUN pip install mkdocs-glightbox
```

Comment:
Why can't we use the docs section in the setup.py file?

Reply:
The base image of Dockerile.documentation is just a image which contains all necessary files to build and deploy mkdocs with material theme, as we are using mkdocs-glightbox plugin we have to install in our image.

setup.py allow us to handle the whole project dependencies, but unfortunately you can not install just the doc dependencies, `using pip install -e ".[doc]`  it will install all StartLeft dependencies rather than just documentation, so in fact we have two images with same components.

This way allow us to provide a small image containing just the docs files and the docs server.

Finally we have to check if this image give us some facilities rather than runs docs locally with docker as we are not using this image to publish our docs in anywhere.